src/masquerade/utils.py

```diff
@@ -14,3 +19,32 @@ def cleanse_text(text):
         return text
 
     return text.strip().replace('"', "").replace("'", "")
+
+
+def get_request_param(request, param_name):
+    """get the parameter from the request checking both url params and form data"""
+    if param_name in request.args:
+        return request.args.get(param_name)
+
+    return request.form.get(param_name)
```

Comment:
**question**: Should we log what params aren't in the request or is that not useful?

Reply:
punted to #179